### ISSUE TITLE
Laravel 5.2 Issue?

### ISSUE THREAD

**Issue by sunscreem** [davejamesmiller/laravel-breadcrumbs#110]
I've just upgraded an existing project to 5.2 and I'm getting the following:

```
ErrorException in helpers.php line 466:
htmlentities() expects parameter 1 to be string, object given (View: C:\wamp2\www\fireworx\laravel\vendor\davejamesmiller\laravel-breadcrumbs\views\bootstrap3.blade.php) (View: C:\wamp2\www\fireworx\laravel\vendor\davejamesmiller\laravel-breadcrumbs\views\bootstrap3.blade.php) (View: C:\wamp2\www\fireworx\laravel\vendor\davejamesmiller\laravel-breadcrumbs\views\bootstrap3.blade.php)
```

Before I started digging into it - I wonder if this is because something needs updated in the package for 5.2?

**Comment by davejamesmiller** [comment]
I haven't yet upgraded myself, but I can't see anything in the upgrade notes that is likely to cause it.

**Comment by sunscreem** [comment]
Ok - I'll hang fire debugging until you've had chance to have a play yourself :)

**Comment by davejamesmiller** [comment]
OK. Just so you know that probably won't be for several weeks, so if you are able to track it down in the meantime please let me know.

**Comment by sunscreem** [comment]
I found the problem. Its with my code.

In my `app/breadcrumbs.php`:

```
        $breadcrumbs->push($category->parentName, url($category->parentUrl));
```

Turns out my `$category->parentUrl` was returning null, so I was (mistakenly) pushing a "blank" breadcrumb.

In 5.1 `url(null)` returned `null` which meant your code, handily, just skipped that blank breadcrumb.

However - this change in 5.2:

>The url() helper function now returns a `Illuminate\Routing\UrlGenerator` instance when no path is provided.

.. meant it all went horribly pete tong.
